sel/src/Sel/SecretKey/Cipher.hs

```diff
@@ -255,16 +255,16 @@ nonceToHexByteString (Nonce nonceForeignPtr) =
 -- | A ciphertext consisting of an encrypted message and an authentication tag.
 --
 -- @since 0.0.1.0
-data Hash = Hash
+data Ciphertext = Ciphertext
```

Comment:
Do we prefer `Ciphertext` or `CipherText`?

I noticed a little bit of inconsistency:

In `Sel.PublicKey.Cipher` and `Sel.SecretKey.Stream`, we have two types named `CipherText`. However, in `Sel.SecretKey.Stream`, there are functions where this capitalization is not consistent: `ciphertextFromHexByteString`, `ciphertextToBinary`, etc.

Just let me know which way you guys prefer and I can make the changes throughout the project, if you like.

Reply:
I've personally always used "ciphertext" as a single word but, of course, I'd like to comply with the API that you prefer.

Reply:
Wikipedia says `Ciphertext`, let's stick with it!

Reply:
Sounds good. 

I'm thinking to follow up with another PR to change `CipherText` to `Ciphertext` in other modules. 

But, if you prefer it to be done in this PR, I'll just append the commits to this branch.

Reply:
I'd rather you did in this PR, at least there's one place to look at. :)

Reply:
Done